src/quemb/molbe/chemfrag.py

```diff
@@ -445,28 +465,28 @@ class PurelyStructureFragmented:
     """
 
     #: The full molecule
-    mol: Final[Mole]
+    mol: Final[Mole] = field(eq=cmp_using(are_equal))
```

Comment:
Here we define that the `__eq__` method of `Fragmented`  uses `are_equal` for comparing two `Mole` objects.

Reply:
discussed in detail below